### PR TITLE
校对 2.9-3.4

### PR DESCRIPTION
**Pull request by ykla**
校对 2.9-3.4

### CODE REVIEW

**Review by das-rob3rt — CHANGES_REQUESTED** [review]

File: di-2-zhang-an-zhuang-freebsd/2.10.-gu-zhang-pai-chu.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
 # 2.10.故障排除
 
-这一节涵盖了基本安装的故障排除，例如人们所报告的常见问题。
+这一节涵盖了基本安装的故障排除，例如人们已经报告的常见问题。
 
-检查 FreeBSD 版本的硬件说明 (<https://www.freebsd.org/releases/>) 文件，以确定硬件是否被支持。如果硬件被支持，但出现了卡住或其他问题，请按照配置 FreeBSD 内核中的说明建立一个定制的内核，以增加对 GENERIC 内核中不存在的设备的支持。默认内核假定大多数硬件设备在出厂时都是默认配置的，包括 IRQ、I/O 地址和 DMA 通道。如果硬件已经被重新配置过，自定义的内核配置文件可以告诉 FreeBSD 在哪里找东西。
+检查 FreeBSD 版本的硬件说明 (<https://www.freebsd.org/releases/>) 文件，以确定是否支持该硬件。如果硬件被支持，但出现了卡住或其他问题，请按照[配置 FreeBSD 内核](https://docs.freebsd.org/en/books/handbook/kernelconfig/index.html#kernelconfig)中的说明构建一个定制的内核，以增加对 **GENERIC** 内核中不存在的设备的支持。默认内核假定大多数硬件设备的参数都是出厂预置的，包括 IRQ、I/O 地址和 DMA 通道。如果硬件被修改过，自定义的内核配置文件可以告诉 FreeBSD 在哪里找到这些硬件。
```
[the remark this answers is not in the document]
Comment (ykla) [reply di-2-zhang-an-zhuang-freebsd/2.10.-gu-zhang-pai-chu.md:5]: Check the Hardware Notes (https://www.freebsd.org/releases/) document for the version of FreeBSD to make sure the hardware is supported. If the hardware is supported and lock-ups or other problems occur, build a custom kernel using the instructions in Configuring the FreeBSD Kernel to add support for devices which are not present in the GENERIC kernel. The default kernel assumes that most hardware devices are in their factory default configuration in terms of IRQs, I/O addresses, and DMA channels. If the hardware has been reconfigured, a custom kernel configuration file can tell FreeBSD where to find things.

File: di-2-zhang-an-zhuang-freebsd/2.10.-gu-zhang-pai-chu.md [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
->**注意：一些安装问题可以通过更新各种硬件组件上的固件来避免或改善，其中最明显的是主板。主板固件通常被称为 BIOS。大多数主板和计算机制造商都有一个升级和升级信息的网站。**
+>**注意**
 >
->**制造商通常建议不要升级主板 BIOS，除非有很好的理由这样做，如关键更新。升级过程可能出错，导致 BIOS 不完整，电脑无法使用。**
+>一些安装问题可以通过更新各种硬件组件上的固件来避免或缓解，其中最明显的是主板。主板固件通常被称为 BIOS。大多数主板和计算机制造商都有一个升级和升级信息的网站。
+>
+>除非是必要的关键更新，否则制造商通常不建议升级主板 BIOS。更新过程可能出错，导致 BIOS 不完整以至于电脑无法使用。
 
-如果系统在启动过程中探测硬件时挂起，或者在安装过程中表现得很奇怪，ACPI 可能是罪魁祸首。FreeBSD 在 i386 和 amd64 平台上广泛使用了系统 ACPI 服务，如果在启动过程中检测到它，就可以帮助系统配置。不幸的是，ACPI 驱动程序和系统主板及 BIOS 固件中仍然存在一些错误。ACPI 可以通过在第三阶段启动加载器中设置`hint.acpi.0.disabled`提示来禁用。
+如果系统在启动过程中探测硬件时挂起，或者在安装过程中表现得很奇怪，可能罪魁祸首是 ACPI。FreeBSD 在 i386 和 amd64 平台上广泛使用了系统的 ACPI 服务，如果在启动过程中检测到它，就可以帮助配置系统。不幸的是，ACPI 驱动程序和系统主板及 BIOS 固件中仍然存在一些问题。可以通过在第三阶段启动加载器中设置 `hint.acpi.0.disabled` 提示来禁用 ACPI：
```
[the remark this answers is not in the document]
Comment (ykla) [reply di-2-zhang-an-zhuang-freebsd/2.10.-gu-zhang-pai-chu.md:13]: If the system hangs while probing hardware during boot, or it behaves strangely during install, ACPI may be the culprit. FreeBSD makes extensive use of the system ACPI service on the i386 and amd64 platforms to aid in system configuration if it is detected during boot. Unfortunately, some bugs still exist in both the ACPI driver and within system motherboards and BIOS firmware. ACPI can be disabled by setting the hint.acpi.0.disabled hint in the third stage boot loader:

File: di-2-zhang-an-zhuang-freebsd/2.9.-wang-luo-jie-kou.md [hunk 3]
```diff
@@ -1,91 +1,93 @@
...
-如果选择了以太网接口，安装程序将跳到选择 IPv4 网络中显示的菜单。如果选择了无线网络接口，系统将转而扫描无线接入点。
+如果选择了以太网卡，安装程序将跳到[选择 IPv4 网络](https://docs.freebsd.org/en/books/handbook/book/#bsdinstall-configure-net-ipv4)中显示的菜单。如果选择了无线网卡，系统将转而扫描无线接入点：
 
 ![](../.gitbook/assets/50.png)
 
-**Figure 50. Scanning for Wireless Access Points**
+***图 50. 扫描无线接入点***
 
-无线网络是由服务集标识符（SSID）来识别的，这是给予每个网络的一个简短而独特的名称。在扫描过程中发现的 SSID 会被列出，然后是对该网络可用的加密类型的描述。如果所需的 SSID 没有出现在列表中，选择`Rescan`以再次扫描。如果想要的网络仍然没有出现，请检查天线连接是否有问题，或者尝试将电脑移到离接入点更近的地方。每次改变后都要重新扫描。
+无线网络是通过服务集标识（SSID）来识别的，这是一个简短而独特的名称。在扫描过程中会列出发现的 SSID，然后会对该网络可用的加密类型进行说明。如果所需的 SSID 没有出现在列表中，可选择 **Rescan** 再扫描一次。如果想要的网络仍然没有出现，请检查天线的连接是否正常，或尝试将电脑移到离接入点更近的地方，然后再扫描一次。
```
[the remark this answers is not in the document]
Comment (ykla) [reply di-2-zhang-an-zhuang-freebsd/2.9.-wang-luo-jie-kou.md:17]: 他想说的是天线。

File: di-3-zhang-freebsd-ji-chu/3.3.-yong-hu-he-ji-ben-zhang-hu-guan-li.md [hunk 4]
```diff
@@ -101,28 +103,29 @@ shell 为用户提供了与系统交互的默认环境。有许多不同种类
 >%
 >```
 
-内置的 su(1)框架对于只有一个系统管理员的单一系统或小型网络来说效果很好。另一个选择是通过二进制包或 ports 安装`security/sudo`。这个软件提供活动记录，并允许管理员配置哪些用户可以作为超级用户运行哪些命令。
+内置的 [su(1)](https://www.freebsd.org/cgi/man.cgi?query=su&sektion=1&format=html) 框架对于只有一个系统管理员的单一系统或小型网络来说效果很好。另一个选择是通过软件包或 ports 安装 [security/sudo](https://cgit.freebsd.org/ports/tree/security/sudo/pkg-descr)。这个软件提供活动记录，并允许管理员配置哪些用户可以作为超级用户运行哪些命令。
```
[the remark this answers is not in the document]
Comment (ykla) [reply di-3-zhang-freebsd-ji-chu/3.3.-yong-hu-he-ji-ben-zhang-hu-guan-li.md:106]: The built-in [su(1)](https://www.freebsd.org/cgi/man.cgi?query=su&sektion=1&format=html) framework works well for single systems or small networks with just one system administrator. An alternative is to install the [security/sudo](https://cgit.freebsd.org/ports/tree/security/sudo/pkg-descr) package or port. This software provides activity logging and allows the administrator to configure which users can run which commands as the superuser.

File: di-3-zhang-freebsd-ji-chu/3.2.-xu-ni-kong-zhi-tai-he-zhong-duan.md [hunk 5]
```diff
@@ -1,34 +1,35 @@
...
 ## 3.2.1.虚拟终端
 
-虽然可以用系统控制台来与系统进行交互，但在 FreeBSD 系统中通过键盘的命令行工作的用户通常会转而登录到一个虚拟控制台。这是因为系统信息被默认配置为显示在系统控制台。这些信息会出现在用户正在处理的命令或文件上，使用户难以集中精力处理手头的工作。
+虽然可以用系统控制台来与系统进行交互，但通过键盘的命令行工作的 FreeBSD 系统用户通常会转而登录到一个虚拟控制台。这是因为系统信息被默认配置为显示在系统控制台。这些信息会在用户正在处理的命令或文件时不断出现，使用户难以集中精力处理手头的工作。
 
-默认情况下，FreeBSD 被配置为提供几个虚拟控制台用于输入命令。每个虚拟控制台都有自己的登录提示和 shell，很容易在虚拟控制台之间切换。这实际上提供了相当于在图形环境中同时打开几个窗口的命令行。
+FreeBSD 预置了几个虚拟控制台用于输入命令。每个虚拟控制台都有自己的登录提示和 shell，且很容易在不同的虚拟控制台之间切换。这实际上提供了相当于在图形环境中同时打开了多个窗口的命令行。
 
-FreeBSD 保留了从`Alt+F1`到`Alt+F8`的组合键，用于在虚拟控制台之间切换。使用`Alt+F1`切换到系统控制台（ttyv0），`Alt+F2`访问第一个虚拟控制台（ttyv1），`Alt+F3`访问第二个虚拟控制台（ttyv2），以此类推。当使用 Xorg 作为图形控制台时，该组合变为`Ctrl+Alt+F1`，以返回到基于文本的虚拟控制台。
+FreeBSD 保留了从 `Alt+F1` 到 `Alt+F8` 的组合键，用于在虚拟控制台之间切换。使用 `Alt+F1` 切换到系统控制台（**ttyv0**），`Alt+F2` 访问第一个虚拟控制台（**ttyv1**），`Alt+F3` 访问第二个虚拟控制台（**ttyv2**），以此类推。当使用 Xorg 作为图形控制台时，该组合变为 `Ctrl+Alt+F1`，以返回到基于文本的虚拟控制台。
```
[the remark this answers is not in the document]
Comment (ykla) [reply di-3-zhang-freebsd-ji-chu/3.2.-xu-ni-kong-zhi-tai-he-zhong-duan.md:26]: The key combinations Alt+F1 through Alt+F8 have been reserved by FreeBSD for switching between virtual consoles. Use Alt+F1 to switch to the system console (ttyv0), Alt+F2 to access the first virtual console (ttyv1), Alt+F3 to access the second virtual console (ttyv2), and so on. When using Xorg as a graphical console, the combination becomes Ctrl+Alt+F1 to return to a text-based virtual console.

[the remark this answers is not in the document]
Comment (ykla) [reply di-3-zhang-freebsd-ji-chu/3.2.-xu-ni-kong-zhi-tai-he-zhong-duan.md:26]: 第1 个tty 是GUI（图形桌面会话）。

File: di-2-zhang-an-zhuang-freebsd/2.10.-gu-zhang-pai-chu.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (das-rob3rt) [reply di-2-zhang-an-zhuang-freebsd/2.10.-gu-zhang-pai-chu.md:5]: > The default kernel assumes that most hardware devices are in their factory default configuration in terms of IRQs, I/O addresses, and DMA channels

默认的 FreeBSD 内核假定大多数硬件的中断请求、I/O 地址、DMA 通道的配置使用厂家的默认配置

> If the hardware has been reconfigured, a custom kernel configuration file can tell FreeBSD where to find things

如果你的硬件被魔改过，请修改内核配置文件来告诉 FreeBSD 如何应对这些特殊的硬件

File: di-2-zhang-an-zhuang-freebsd/2.10.-gu-zhang-pai-chu.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (das-rob3rt) [reply di-2-zhang-an-zhuang-freebsd/2.10.-gu-zhang-pai-chu.md:13]: > Unfortunately, some bugs still exist in both the ACPI driver and within system motherboards and BIOS firmware. ACPI can be disabled by setting the hint.acpi.0.disabled hint in the third stage boot loader

The "some bugs" right here is apparently not guaranteed to occur, it's rather something that CAN happen. However, the translation makes it sound like it MUST happen.

**Pull request merged**